index.html

```diff
@@ -1,8 +1,11 @@
 <!DOCTYPE html>
-<html>
+<html class="md-scrollbar">
```

Comment:
It was done according to [this](https://github.com/marcosmoura/vue-material/issues/46) and it works. I agree other approaches mentioned in the issue look better but I had trouble making them work so I stick to this one. I could not find mention about this in official documentation.